instruqt-tracks/nomad-job-placement/track.yml

```diff
@@ -143,8 +143,8 @@ challenges:
     `
     constraints = ["tag==service"]
     `<br>
-
-    If you look back at the "webapp.nomad" job specification, on line 33 you will see that the same service tag was specified in the tags section of the registration of the web app with Consul.
+    
+    If you look back at the "webapp.nomad" job specification, on line 32 you will see that the same service tag was specified in the tags section of the registration of the web app with Consul.
```

Comment:
Should this be line 31?  I was counting inside the script but got 31 twice.

Reply:
OK, with the changes of the network stanza, removing mbits, and the addition of the "ports = ["http"]" the line with     "traefik.tags=service" is on line 34, so I made an updated commit for the corrected number.  Shame because 33 is such a powerful number and we're off by one!